openhands/runtime/impl/remote/remote_runtime.py

```diff
@@ -361,49 +325,53 @@ def run_action(self, action: Action) -> Observation:
                 return NullObservation('')
             action_type = action.action  # type: ignore[attr-defined]
             if action_type not in ACTION_TYPE_TO_CLASS:
-                return FatalErrorObservation(
-                    f'[Runtime (ID={self.runtime_id})] Action {action_type} does not exist.'
+                return ErrorObservation(
```

Comment:
ditto - we should raise exception here instead error observation

Reply:
fixed!